client/src/pages/game/GamePrompt.js

```diff
@@ -1,10 +1,28 @@
-import React from "react";
+import React, { useState, useEffect, useContext } from "react";
 import { Box, Container, Grid, Typography } from "@material-ui/core";
 import TimerIcon from "@material-ui/icons/Timer";
-
+import { AppContext } from "../../App";
 import { TEAM_ROLE } from "../game_lobby/team_select/TeamPresets";
 
 function GamePrompt({ gameState, player }) {
+  const { gameIO } = useContext(AppContext);
+
+  const [timer, setTimer] = useState(45);
+
+  let countDown = null;
+
+  gameIO.state.io.on("start timer", () => {
+    clearTimeout(countDown);
+    setTimer(45)
+  });
```

Comment:
This should be in its own useEffect():
useEffect(() => {
  gameIO.state.io.on("start timer", () => {
    clearTimeout(countDown);
    setTimer(45)
  });
}, []); // [] means set the listener only once on mount